fuse_constraints/include/fuse_constraints/normal_delta_pose_3d_cost_functor.h

```diff
@@ -68,6 +68,8 @@ namespace fuse_constraints
 class NormalDeltaPose3DCostFunctor
 {
 public:
+  FUSE_MAKE_ALIGNED_OPERATOR_NEW();
```

Comment:
Here you do need this, but it's curious that the residual weighting matrix is fixed and not variable here https://github.com/locusrobotics/fuse/pull/65/files#diff-0f64d467f76db777867c720f4bae56f7R94, as opposed to the `pose_2d` case above.

Reply:
For whatever reason we do not currently support partial measurements in the 3D case. Made a note to look into this as well.